api-tests/features/repeat-fraud-check/p2-repeat-fraud-check.feature

```diff
@@ -50,16 +50,6 @@ Feature: Repeat fraud check journeys
     Then I get a 'dcmaw' CRI response
     When I submit 'kenneth-changed-family-name-driving-permit-valid' details to the CRI stub
     Then I get a 'page-dcmaw-success' page response with context 'coiNoAddress'
-    When I submit a 'next' event
```

Comment:
I've cut this down (and the one on line 90) instead of just removing it to test the routing in `repeat-fraud-check.yaml` and the bits after after tested in the test above it since we've now merged the journeys together